service/common/src/test/java/org/apache/polaris/service/catalog/io/FileIOFactoryTest.java

```diff
@@ -198,7 +198,7 @@ public void testLoadFileIOForCleanupTask() {
     // 1. BasePolarisCatalog:doCommit: for writing the table during the creation
     // 2. BasePolarisCatalog:doRefresh: for reading the table during the drop
     // 3. TaskFileIOSupplier:apply: for clean up metadata files and merge files
-    Mockito.verify(testServices.fileIOFactory(), Mockito.times(3))
```

Comment:
nit: We need to update the comment to reflect the change. The first method call now comes from initializing the default catalog file IO.